R/ard_categorical.R

```diff
@@ -196,7 +198,13 @@ ard_categorical <- function(data,
 #'   denominator = "cell",
 #'   statistic = list(ARM = list(tabulation = c("N")))
 #' )
-.calculate_tabulation_statistics <- function(data, variables, by, strata, denominator, statistic, call = parent.frame()) {
+.calculate_tabulation_statistics <- function(data,
+                                             variables,
+                                             by,
+                                             strata,
+                                             denominator,
+                                             statistic,
+                                             call = get_cli_abort_call()) {
```

Comment:
Same comment as above regarding the `call` argument. Do you think we can just leave it out?